test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt

```diff
@@ -426,12 +426,14 @@ sun/security/krb5/ktab/FileKeyTab.java https://github.com/ibmruntimes/openj9-ope
 sun/security/krb5/ktab/KeyTabIndex.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 sun/security/krb5/runNameEquals.sh https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 sun/security/krb5/auto/Cleaners.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x
-java/lang/ProcessBuilder/JspawnhelperWarnings.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
+java/lang/ProcessBuilder/JspawnhelperWarnings.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 java/lang/String/CompactString/NegativeSize.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
-java/lang/Thread/virtual/stress/GetStackTraceALotWhenPinned.java#id0 https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
+java/lang/Thread/virtual/stress/GetStackTraceALotWhenPinned.java#id0 https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 java/security/Security/ConfigFileTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x
 javax/crypto/CipherSpi/ResetByteBuffer.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-s390x
 java/security/Security/ConfigFileTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
+java/security/KeyStore/PKCS12/CheckNullDefault.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,aix-all,linux-ppc64le,linux-s390x
+java/util/stream/GathererShortCircuitTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,aix-all,linux-ppc64le,linux-s390x
```

Comment:
Same as above.

Reply:
I dont see this one from the normal runs